portfolio/src/main/webapp/c.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Chinese</title>
+        <link rel="stylesheet" href="style.css">
+        <script src = "./js/index.js"> </script>
+    </head>
+
+    <body id = "content">
+        <!--nav var-->
+		<div id="navbar" class="navbar">
```

Comment:
The formatting looks a little messed up. Shift to the left by 1 tab. It is easier to read when it is formatted well.

Reply:
Done